scripts/compile_protos.mk

```diff
@@ -0,0 +1,74 @@
+PROTOC_VERSION := 26.1
+SWIFT_PROTOBUF_VERSION := 1.25.2
```

Comment:
I'm curious if there could be a way to have some automatic version update from time to time. As I'm afraid we will forget about these version for long time.

Reply:
I couldn't find a quick way to do that, however, I created a task to research if we can keep it in sync across SDK-s. 
https://linear.app/swmansion/issue/FCE-763

@PiotrWodecki suggested a docker image that could be shared.